### ISSUE TITLE
Nerf the Quarg Skylance

### ISSUE THREAD

**Issue by brendanjones** [endless-sky/endless-sky#8608]
### Problem Description

Ignoring the low-availability super-weapons like the Pug Gridfire Turret, the Quarg Skylance is simply too powerful relative to other weapons. I don't agree with [this](https://github.com/endless-sky/endless-sky/issues/267#issuecomment-165330304):

> Basically, once you're strong enough to plunder the Quarg, nothing is going to be a challenge for you - you've won the game. 

Given you can plunder Quarg ships within a few hours of starting a character. You shouldn't be able to reach endgame conditions within hours, and the endgame shouldn't be so mindlessly easy.

Granted, yes it's possible to have some fun with other builds that exploit ships in other ways, like massing ion cannons, flamethrowers, or the various Korath weapons, but if you're going for pure DPS the Skylance is unmatched. It's possible to put 2 Skylances on most larger ships without skimping on other outfits, for 6,000 DPS. On the largest ships like a Kar Ik Vot 349 you can have 3-4 Skylances for 9,000-12,000 DPS. That means a single one of your ships can take down any other ship in a couple of seconds. Do we not have a problem with that? Isn't this a bit close to being a [forklift](https://www.youtube.com/watch?v=-P2VrdrlWVU)?

Other Quarg outfits aren't so overpowered. They're often a bit better than the purchasable tech of other races (Antimatter Core & Nanotech battery), sometimes about the same (Quantum Shield Generator & Anti-Missile Turret) and sometimes actually worse (Gravitron Thruster & Steering). However based on damage-per-second per mass, the Skylance is about 5x more powerful than any other turret. That's a massive difference.

Notice I didn't say 'nerf the Quarg', just the Skylance. The Quarg are meant to be powerful, and should maintain that superiority. Given the suggested nerfs below, the Quarg ships could be given more Skylances. Also, as suggested [here](https://github.com/endless-sky/endless-sky/issues/267#issuecomment-159988298), more Quarg ships should appear when you attack them and when you have low reputation with them, to make farming Quarg ships significantly more difficult. But that's for a separate issue.

### Related Issue Links

https://github.com/endless-sky/endless-sky/issues/267
Actually surprised there aren't more - please add if you know of any.

### Desired Solution

Damage: drop it from 3000/2400 DPS (shield/armor) to 2000/1800. This still keeps the Skylance 2-3 times more powerful than the most powerful turrets from other races.

### Alternative Approaches

There are more attributes we could play with, in order of how much I like the idea (most-liked first):

- Firing energy/second: increase it from 1800 to 2400. Wardragons and Skylarks have massive energy generation and storage so this wouldn't affect them, while limiting the usage of Skylances on ships using non-Quarg energy tech.
- Mass/outfit space: increase it from 60 to 80
- Turret mounts required: could increase this from 1 to 2. This would limit the number of Skylances you could put on a ship.
- Turn rate: It's currently much higher than almost every other turret, so could drop it from 240 to, say 200.
- Range: I wouldn't touch this, I like the differentiation this offers over other turrets and it feels like it matches Quarg technological prowess. Could even increase it, especially if the DPS is being lowered.
- firing heat and inaccuracy: I wouldn't touch these, as they match Quarg tech (efficient and advanced)

### Additional Context

<img width="263" alt="Screenshot 2023-03-31 at 12 16 55" src="https://user-images.githubusercontent.com/16049594/229095067-89de1fb2-04b3-4b6e-a23c-5f04ac5ca2d6.png">

**Comment by Hecter94** [comment]
I think the issue here is less the Skylance and more: 

> Given you can plunder Quarg ships within a few hours of starting a character. You shouldn't be able to reach endgame conditions within hours.

This seems like the problem. Skylances are an endgame outfit; they *should* be extremely powerful. If they aren't difficult to obtain, they should be made more difficult to obtain. 
Perhaps something like:
https://github.com/endless-sky/endless-sky/issues/3346 Could be expanded to include things like Skylances as being unplunderable without fighting
https://github.com/endless-sky/endless-sky/issues/2948 could restrict plundering as well as capturing without specialized tools.
https://github.com/endless-sky/endless-sky/issues/871 could make it more difficult for the player to plunder stuff while hostile Quarg are attacking them.

The easiest solution would probably be to make most T3 outfits unplunderable, requiring the player to be able to capture the ship in order to obtain them, which helps to function as a gate to extremely quick progression. 

We could also look at what in the game allows the player to fight and disable endgame ships so soon after starting and fix whatever that is. Are the Quarg ships getting disabled via heat or ion? Perhaps they need resistance. Are they getting swarmed by a massive fleet and losing to numbers? They may need to prioritize their safety more and jump away sooner. Or getting such a large fleet should be made more difficult to do.

**Comment by Saugia** [comment]
Keep in mind, all Quarg content right now is currently placeholder and subject to change. Changes have already been discussed many times, and one of those is to lower the power of a Skylance but add stronger and "weaker" weapons to the Quarg so they aren't just using one thing. @Arachi-Lover would know more about this, but for the relevance of this issue, assume the current Skylance will be changed.

**Comment by Arachi-Lover** [comment]
Not much to add that hasn't been said. It's just a matter of the Quarg needing to be rebalanced and expanded upon with more ships and outfits really.
Skylance's gonna be nerfed, they'll get stronger stuff, all that.
I do support the thought of making Quarg or really just T3 outfits in general unplunderable, and look on how to make it harder to disable their ships.

**Comment by Hurleveur** [comment]
You can use the 621 (I think thats the one) and get up to 9 skylances I believe, thats how I got the two arfectas in my one flagship playtrough

**Comment by brendanjones** [comment]
> I think the issue here is less the Skylance and more:
> Given you can plunder Quarg ships within a few hours of starting a character. You shouldn't be able to reach endgame conditions within hours.
> This seems like the problem. Skylances are an endgame outfit; they should be extremely powerful.

In general I agree, but I still think the Skylance needs nerfing in some way. Even if it's to limit the number of lances you can have on one ship by increasing the outfit mass and/or increasing the firing energy. As I said in my original post, I just don't think there should be one weapon that does 5-6 times more DPS than almost every other weapon. Even a Gridfire Turret takes 23 tonness more mass, 600 more energy to fire per second, and does about 60% of the DPS, and you can only get a few of those (okay, you get way more range).

> The easiest solution would probably be to make most T3 outfits unplunderable, requiring the player to be able to capture the ship in order to obtain them, which helps to function as a gate to extremely quick progression.

That certainly delays plunder of Quarg outfits en masse by at least a day of playtime. Once you've got one of the Korath ships with high crew count then it's pretty easy to capture Skylarks. But without the ability to quickly plunder Sklylances it takes way longer to amass a fleet that can comfortably take down Quarg ships and defend your min/maxed slow-ass Korath ship while it tries to board and capture a Skylark. So yes, I'd be on board with this, it's a simple way to slow plunder of Quarg ships without introducing breaking changes nor breaking the lore of the game. But as you @hector94 said, other solutions would still need to be looked at.

Thanks for the responses everyone.

**Comment by brendanjones** [comment]
> It may have dps but it lacks range a lot.

@Hurleveur The range is longer than most other weapons; the only time it's an issue is if you're trying to take on Korath ship(s) without escorts, and they're keeping you at range with their repulsor weapons. Otherwise the fact that you can make [capital ships go as fast as fighters](https://github.com/endless-sky/endless-sky/issues/5271) means that the lack of range isn't really an issue.

**Comment by Hecter94** [comment]
> Once you've got one of the Korath ships with high crew count then it's pretty easy to capture Skylarks. 

Making T3 outfits unplunderable, alongside https://github.com/endless-sky/endless-sky/pull/8506 could prevent the capture of Quarg ships and therefore looting of Skylances behind any arbitrary progression level we want.

**Comment by Quantumshark** [comment]
I really don't like the idea of using `unplunderable` as a substitute for proper game balance, and adding it to skylances would also be inconsistent with the current lore about the Heliarchs having a small stash of them (also, if the Quarg have the ability to make their outfits impossible to plunder, it doesn't make sense that they wouldn't do the same with their Jump Drives, so you've got an even bigger problem there.

The skylance and antimatter core are both currently too strong for T3 and _should_ both be nerfed significantly when the Quarg are eventually deplaceholderised, at which point they shouldn't represent more of a step up over Korath/Wanderer outfits (T2) than Korath/Wanderer outfits do over human technology.

**Comment by Hecter94** [comment]
> I really don't like the idea of using `unplunderable` as a substitute for proper game balance, 

`balance` is both stats and availability. You can only consider one with the other. Tweaking availability is just as valid for balancing purposes as tweaking stats is.

**Comment by Quantumshark** [comment]
> `balance` is both stats and availability. You can only consider one with the other. Tweaking availability is just as valid for balancing purposes as tweaking stats is.

Fair point, but it's still a lore issue, and the skylance and antimatter core still need to be nerfed either way.

**Comment by Hecter94** [comment]
I'll allow Arachi to comment on the lore "issue", but a couple of things to consider:
1. To the best of my knowledge, it's never been **confirmed** that the Heliarch does have Skylances. Sure, there are rumors, but no in-game confirmation that they do. 
2. Even if they do, they've had far more interaction with the Quarg than the player ever would, giving them ample opportunity to capture some of their tech via avenues unavailable to the player. 
3. There is a logic in keeping T3 stats quite high as they function as the ceiling under which all other races we're adding have to stay. We've already decided that T3 tech will grow as strong as it needs to give us enough room in the T1/T2 levels to build the rest of the game.

**Comment by Arachi-Lover** [comment]
As far as the Heliarchs having Quarg stuff goes, yes they do have an outfit stash; I don't think that gameplay-wise Skylances being unplunderable would go against that necessarily.
If it is a concern though, with the aforementioned #8506 (whenever that gets in) it could just be made the case that the required equipment for looting Quarg ships is gated by the Heliarchs, so that it does match in gameplay terms.

**Comment by ThrawnCA** [comment]
I'll just point out that the Quarg will now punish you for having their outfits, which means if you do plunder them, you're liable to have trouble completing some of the storylines.

**Comment by brendanjones** [comment]
I don't think this is leading to any specific changes, so closing to clean up the issues list. I still think the Skylance DPS should be dropped a bit without waiting for other changes to the Quarg, but it looks like there's no agreement for that.

**Comment by TomGoodIdea** [comment]
It's been rebalanced with #11102 anyways.

**Comment by brendanjones** [comment]
Wow. WOW. That is very cool, thanks for linking that. Amazing to see such awesome new content being developed, and all the work that goes into it.
